apps/client/src/features/rundown/event-block/RundownIndicators.module.scss

```diff
@@ -0,0 +1,25 @@
+.indicators {
+  font-size: calc(1rem - 5px);
+  position: absolute;
+  top: -1em;
+  z-index: 2;
+  margin: 0 20%;
+  width: 60%;
+
+  display: flex;
+  gap: 0.5rem;
+}
+
+@mixin indicator($bg-colour) {
+  padding: 0 0.5rem;
+  border-radius: 2px;
+  background-color: $bg-colour;
+}
+
+.delay {
+  @include indicator($ontime-delay);
+}
+
+.gap {
+  @include indicator($blue-500);
+}
```

Comment:
I find the position of the indicators a bit random/off center
what do you think about pinning the gap indicator to the right with the other indicators and the delay inline with the the start timer filed that it is describing
![image](https://github.com/cpvalente/ontime/assets/19875125/b2721cf4-79e6-4461-97c3-d97111c7b2b3)

```css
.indicators {
  font-size: calc(1rem - 5px);
  position: absolute;
  top: -1.15em;
  z-index: 2;
  margin: 0 8em;
  width: 85%;
  text-align: center;
}

@mixin indicator($bg-colour) {
  padding: 0 0.5rem;
  border-radius: 2px;
  background-color: $bg-colour;
}

.delay {
  @include indicator($ontime-delay);
  width: max-content;
  position: absolute;
  left: 0;
}

.gap {
  @include indicator($blue-500);
  width: max-content;
  position: absolute;
  right: 0;
}

```

Reply:
I am not sure I agree. since these labels are dynamic, having them in a row improves their visibility.
Otherwise I fear users would try to find hidden meanings int their position, which is untrue.

Maybe you are correct that their current position seem to indicate that they belong to the input field. I just didnt want to have them over the buttons 😓 

Since this PR is so large, I suggest we roll with this and revisit styles in beta